bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```
@@ -403,11 +378,12 @@ spec:
                 path: admin-password
         {{- end }}
         {{- if .Values.pgpool.tls.enabled }}
-        - name: raw-certificates
-          secret:
-            secretName: {{ include "postgresql-ha.tlsSecretName" . }}
         - name: pgpool-certificates
-          emptyDir: {}
+          projected:
+            defaultMode: 0600
+            sources:
+              - secret:
+                  name: {{ include "postgresql-ha.tlsSecretName" . }}
```

There is no real need to use a projected volume in this case. I think this piece of code has the same behaviour (I did not test):
```suggestion
          secret:
            defaultMode: 0600
            secretName: {{ include "postgresql-ha.tlsSecretName" . }}
```